src/main/java/org/example/services/BucketService.java

```
@@ -0,0 +1,130 @@
+        ListObjectsV2Result listResult;
+        listResult = s3client.listObjectsV2(req);
+        for (S3ObjectSummary objectSummary : listResult.getObjectSummaries()) {
+            System.out.println(" - " + objectSummary.getKey() + "  " + "(size = " + objectSummary.getSize() + ")");
+            filename = objectSummary.getKey();
+           // String[] splitFilepath = objectSummary.getKey().split("/");
+           // filename = splitFilepath[2];
+            System.out.println(filename);
+            //filename = objectSummary.getKey();
+
+            System.out.format("Downloading %s from S3 bucket %s...\n",
+                    filename, System.getenv().get("S3_BUCKET"));
+            try {
+                S3Object o = s3client.getObject(System.getenv().get(
+                        "S3_BUCKET"), filename);
+                S3ObjectInputStream s3is = o.getObjectContent();
+
+
+                InputStreamReader inputStreamReader = new InputStreamReader(
+                        s3is);
+                BufferedReader bufferedReader = new BufferedReader(
+                        inputStreamReader);
+                System.out.println("NOT AWS FILE HERE");
+                String line;
+                List<String> listOfLines = new ArrayList<>();
+                int counter = 0;
+                while ((line = bufferedReader.readLine()) != null) {
+                    counter++;
+                    // Process the line
+                    System.out.println(line);
+                    fileImportValidator.validateFileImportLine(line, counter);
+                    listOfLines.add(line);
+                }
+                System.out.println(listOfLines);
+
+                result = fileImportDao.importRoles(listOfLines);
+
+            } catch (AmazonServiceException e) {
+                System.err.println(e.getErrorMessage());
+                System.exit(1);
```

Please don't use system.exit(1). These should throw the exceptions and then they can be handled correctly